Admin/AbstractAdmin.php

```diff
@@ -55,7 +55,15 @@
     const CONTEXT_MENU = 'menu';
     const CONTEXT_DASHBOARD = 'dashboard';
 
-    const CLASS_REGEX = '@(?:([A-Za-z0-9]*)\\\)?(Bundle\\\)?([A-Za-z0-9]+?)(?:Bundle)?\\\(Entity|Document|Model|PHPCR|CouchDocument|Phpcr|Doctrine\\\Orm|Doctrine\\\Phpcr|Doctrine\\\MongoDB|Doctrine\\\CouchDB)\\\(.*)@';
+    const CLASS_REGEX =
+        '@
+        (?:([A-Za-z0-9]*)\\\)?        # vendor name / app name
```

Comment:
> Does comment inside regex works on every PHP version?

I learnt that trick many years ago so I'd say yes.

> Can you double-check we have a test about this regex?

When I remove the `y` in Entity, all hell breaks loose, so at least part of the regex is covered.